frontend/src/Judge/components/ShowCard.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import { Button, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+const ButtonContainer = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: '150px'
+`
+
+const ShowCard = props => (
+  <Card>
+    <h2>
+      <Link to={`show/${props.id}`}>{props.name}</Link>
+    </h2>
+    <Col>
+      <dl>
+        <dt>Judging Ends:</dt>
+        <dd>
+          <Moment format='YYYY/MM/DD'>{props.judgingEnd}</Moment>
+        </dd>
+      </dl>
+    </Col>
+    <Col>
+      <ButtonContainer>
+        <Button
+          style={{ cursor: 'pointer' }}
+          tag={Link}
+          to={`judge/vote`} // TODO: Figure out where they go to vote
```

Comment:
This might also be a query param that goes to  <Link to={`/vote?to=${forShow.id}`}>
And also putting in their username or something for the unique shuffle? This is probably a later thing.

Reply:
I'm thinking the URL formats we'll use on the Judge app are:

`/show/<show_id>/` => shows the review page for a show (the list view of all entries and what the judge voted)

`/show/<show_id>/vote/` =>  **(use this here)** Prepares the voting queue for the particular show. Fetches all entries, creates a shuffle order, and then renders the first unvoted entry in the voting interface (updating the URL to the format below). If the voting queue has already been prepared, accessing this route will recalculate the last unvoted entry and go to it.

`/show/<show_id>/vote?entry=<entry_id>` => shows the voting interface w/ the particular entry

Reply:
I agree with bill on this one